src/main/java/koksao/Main.java

```diff
@@ -0,0 +1,92 @@
+package koksao;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Options;
+
+import javax.swing.text.html.Option;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+
+
+    public static void main(String[] args) {
+        Options options = new Options();
+        options.addOption("r", "rates", true, "path to a file");
+
+        CommandLineParser clp = new DefaultParser();
+        try {
+            CommandLine cl = clp.parse(options, args);
+            if (cl.hasOption("r")) {
+                String filePath = cl.getOptionValue("r");
+                args[0] = filePath;
+            } else {
+                System.out.println("Cannot find a file");
+            }
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        Scanner scanner = new Scanner(System.in);
+        String answer = "";
+        String firstCurrency;
+        String secondCurrency;
+
+        while (!answer.equals("Exit")) {
+            System.out.println("What currency you want to convert from?");
+            firstCurrency = scanner.next();
+            System.out.println("On which?");
+            secondCurrency = scanner.next();
+            System.out.println("What amount of money?");
+            int amount = scanner.nextInt();
+
+            String line;
+            String csvSplitBy = ",";
+            try {
+                BufferedReader br = new BufferedReader(new FileReader(args[0]));
```

Comment:
You are reading this file each iteration of the while loop. You could actually pass the file to the constructor of the Currencies class before this loop and initialize currencies there, later on you can just use your class here. Additionally pls try to use some library for reading csv files (opencsv or something)